components/zcash_address/src/kind/unified/address.rs

```diff
@@ -267,9 +294,12 @@ pub mod testing {
         arb_typecodes()
             .prop_flat_map(arb_unified_address_receivers)
             .prop_map(|rs| {
-                let mut items = rs.into_iter().map(Item::Data).collect::<Vec<_>>();
-                items.sort_unstable_by(Item::encoding_order);
-                Address(items)
+                let mut receivers = rs.into_iter().map(Item::Data).collect::<Vec<_>>();
+                receivers.sort_unstable_by(Item::encoding_order);
+                Address {
+                    revision: Revision::R0,
```

Comment:
Given that this strategy doesn't generate metadata, it should be fine to generate an arbitrary revision?